code/modules/admin/topic.dm

```diff
@@ -1748,3 +1748,13 @@
 		if(!paper_to_show)
 			return
 		paper_to_show.ui_interact(usr)
+	else if(href_list["play_music"])
+		if(!check_rights(R_SOUND))
+			return
+
+		var/link_url = locate(href_list["play_music"])
+		if(!link_url)
+			return
+
+		var_internet_sound(usr, link_url)
```

Comment:
this can not play on click there has to be an intermittent step that shows what is to be played and prompts the admin.

Reply:
It still has the standard prompts Play internet sound has, where it prompts the song name, and prompts if they want to show they played it, so it has two points where the input can be canceled and the admin is already prompted before the music plays, it isnt a one click play button.